lib/package.json

```diff
@@ -31,34 +31,33 @@
   "devDependencies": {
     "@types/chai": "^3.4.35",
     "@types/chai-as-promised": "^0.0.30",
-    "@types/core-js": "0.9.36",
     "@types/karma-chai-sinon": "^0.1.5",
-    "@types/mocha": "^2.2.39",
-    "@types/sinon": "^1.16.31",
+    "@types/mocha": "^2.2.40",
+    "@types/sinon": "^1.16.35",
+    "awesome-typescript-loader": "^3.1.2",
+    "babel-core": "^6.24.0",
+    "babel-plugin-es6-promise": "^1.1.1",
+    "babel-preset-env": "^1.2.2",
+    "babel-preset-es2015": "^6.24.0",
```

Comment:
`babel-preset-env` to take care of target. Including `babel-preset-es2015` preset was a mistake.